[Konvoy] airgapped bundles (#4270)
mesosphere/dcos-docs-site: pages/dkp/konvoy/2.2/choose-infrastructure/aws/air-gapped/bootstrap/index.md

Konvoy deploys all cluster lifecycle services to a bootstrap cluster, which deploys a workload cluster. When the workload cluster is ready, move the cluster lifecycle services to the workload cluster, after which the workload cluster manages its own lifecycle.

> 1.  Download the bootstrap docker image on a machine that has access to the tar.

Reviewer: ```suggestion
1.  Download the bootstrap docker image on a machine that has access to this artifact.
```

Konvoy deploys all cluster lifecycle services to a bootstrap cluster, which deploys a workload cluster. When the workload cluster is ready, move the cluster lifecycle services to the workload cluster, after which the workload cluster manages its own lifecycle.

1.  Set an environment variable with the DKP version.
    
    ```bash

>     export DKP_VERSION=v2.2.0

Reviewer 2: In https://github.com/mesosphere/dcos-docs-site/pull/4233#discussion_r840865116, we chose to move away from using a variable like this in the Kommander docs to be consistent with the Konvoy docs 🙈

Author: Its funny cause I wrote that comment too :D

Author: Fixed

    ```

1.  Download the bootstrap docker image.

    ```docker
    curl -O https://downloads.d2iq.com/dkp/$DKP_VERSION/konvoy-bootstrap_$DKP_VERSION.tar
    ```

> 1.  Load the bootstrap Docker image.

Reviewer 3: ```suggestion
Load the bootstrap docker image.
```

> 1.  Load the bootstrap Docker image.

Reviewer 3: ```suggestion
1.  Load the bootstrap docker image.
```
